gpg.yaml

```
@@ -0,0 +1,14 @@
+spack:
+  view: /bootstrap/runner/view
+  
+  config:
+    install_tree:
+      root: /bootstrap/runner/install
+
+
+  packages:
+    all:
+      target: [x86_64]
```

```suggestion
  packages:
    all:
      require:
      - one_of:
        - target=x86_64
        - target=aarch64
        - target=ppc64le
```

Sorry, but this was still wrong if you target multiple platforms.